src/lib/syncable_ledger/syncable_ledger.ml

```diff
@@ -24,13 +24,35 @@ module Query = struct
             (** How many accounts are there? Used to size data structure and
             figure out what part of the tree is filled in. *)
       [@@deriving sexp, yojson, hash, compare]
+
+      let to_latest = function
+        | What_child_hashes addr ->
+            What_child_hashes addr
+        | What_contents addr ->
+            What_contents addr
+        | Num_accounts ->
+            Num_accounts
```

Comment:
why not just `Fn.id` ?

Reply:
Actually, it may not be necessary, as I was ultimately decided to not change the query, and if it worked without having `to_latest` it should still work without it, I'll remove it.